Gekko/DataProviders/BTCTradeUA/TradingPlatformAccessibilityController.swift

```diff
@@ -0,0 +1,41 @@
+//  Created by Aleksandr Saliienko on 4/12/18.
+//  Copyright © 2018 Sergii Mykhailov. All rights reserved.
+//
+
+import Foundation
+
+protocol ChangeServerAccessibilityDelegate {
+    func setupServerErrorView()
```

Comment:
Delegate method should not declare what to do. It should declare what was discovered. 
For example these methods should be called:
func tradingPlatformAccessibilityControllerDidDetectConnectionFailure(_ sender:TradingPlatformController) 
func tradingPlatformAccessibilityControllerDidDetectConnectionRestore(_ sender:TradingPlatformController)